client/src/styles/Desk.scss

```diff
@@ -1,3 +1,21 @@
+.hw-card:not(:last-child) {
+  margin-bottom: 15px;
+}
+
+.hw-qty {
+  margin-left: 0 !important;
+}
+
+.hw-positive {
+  color: #28a745;
+}
+
+.hw-negative {
+  color: #dc3545;
+}
+
+
+
 .hardware-item-user-name {
```

Comment:
would prefer if you could replace `.hardware` with `.hw` for consistency if that's what it stands for

Reply:
this is now an unused class, I removed it and renamed the item class though